test/fixImageSize.js

```diff
@@ -0,0 +1,13 @@
+const sharp = require('sharp');
+const fs = require('fs');
+ 
+exports.fixImage = function fixImage(imageFileName){
+    sharp(imageFileName)
+        .resize({width: 512, height: 512})
+        .toFile(imageFileName+ "1").then(()=>{
+        fs.rename(imageFileName+ "1",imageFileName, (err) => {
+            if (err) throw err;
+             console.log('Rename complete!');
```

Comment:
This is not a super useful console log; we should either remove it, or add more detail (ex filename, size reduction, etc.)